hkdf.go

```diff
@@ -214,6 +227,36 @@ func ExpandHKDFOneShot(h func() hash.Hash, pseudorandomKey, info []byte, keyLeng
 	return out, nil
 }
 
+// ExpandHKDFOneShotTLS13_KDF_Support derives a key from the given hash, key, and context info, and will use
+// the TLS 1.3 KDF if the info parameter is in the expected format.
+func ExpandHKDFOneShotTLS13_KDF_Support(h func() hash.Hash, pseudorandomKey, info []byte, keyLength int) ([]byte, error) {
```

Comment:
Why is "Support" part of this func's name?

Also, this module only uses `_` in an exported name to separate two numbers so far, I'm not sure it's worth breaking the convention here.

```suggestion
// ExpandHKDFOneShotTLS13KDF derives a key from the given hash, key, and context info, and will use
// the TLS 1.3 KDF if the info parameter is in the expected format.
func ExpandHKDFOneShotTLS13KDF(h func() hash.Hash, pseudorandomKey, info []byte, keyLength int) ([]byte, error) {
```

Reply:
See https://github.com/golang-fips/openssl/pull/272#discussion_r2052870493.

Also, `HKDF` should not appear in the name of the function.

Reply:
Removed `HKDF` here: 26ab78edfb2abeb7bc171caa2f6a4dcc1e03198b